src/org/labkey/test/tests/SampleTypeNameExpressionTest.java

```diff
@@ -74,7 +74,7 @@ public class SampleTypeNameExpressionTest extends BaseWebDriverTest
     private static final String PROJECT_NAME = "SampleType_Name_Expression_Test";
 
     private static final String DEFAULT_SAMPLE_PARENT_VALUE = "SS" +
-            EscapeUtil.escapeForNameExpression(TestDataGenerator.randomString(3));
+            EscapeUtil.escapeForNameExpression(TestDataGenerator.randomString(3, "_)(}{")); // '_' is used as delimiter to get batchRandomId and (){} is used to close the defaultValue()
```

Comment:
I just merged my change for this.
I don't really care which one is used, they are both the same.